spk/bazarr/Makefile

```diff
@@ -65,11 +65,8 @@ else
 WHEELS += src/requirements-crossenv-greenlet-v1.txt
 endif
 
-# [numpy] <= 1.22.4 (armv7l)
-ifeq ($(findstring $(ARCH),$(ARMv7L_ARCHS)),$(ARCH))
-WHEELS += src/requirements-crossenv-numpy-armv7l.txt
```

Comment:
It seems that `ARMv7L_ARCHS` are actually part of the unsupported architectures and this line is therefore useless